homeassistant/components/mediaroom/media_player.py

```diff
@@ -136,13 +133,13 @@ def set_state(self, mediaroom_state):
         """Map pymediaroom state to HA state."""
 
         state_map = {
-            State.OFF: STATE_OFF,
-            State.STANDBY: STATE_STANDBY,
-            State.PLAYING_LIVE_TV: STATE_PLAYING,
-            State.PLAYING_RECORDED_TV: STATE_PLAYING,
-            State.PLAYING_TIMESHIFT_TV: STATE_PLAYING,
-            State.STOPPED: STATE_PAUSED,
-            State.UNKNOWN: STATE_UNAVAILABLE,
+            State.OFF: MediaPlayerState.OFF,
+            State.STANDBY: MediaPlayerState.STANDBY,
+            State.PLAYING_LIVE_TV: MediaPlayerState.PLAYING,
+            State.PLAYING_RECORDED_TV: MediaPlayerState.PLAYING,
+            State.PLAYING_TIMESHIFT_TV: MediaPlayerState.PLAYING,
+            State.STOPPED: MediaPlayerState.PAUSED,
+            State.UNKNOWN: None,
```

Comment:
Minor tweak here, `State.UNKNOWN` will now map to `STATE_UNKNOWN` not `STATE_UNAVAILABLE`